### ISSUE TITLE
Using `ini()` without loading the package first can't find lotri

### ISSUE THREAD

**Issue by billdenney** [nlmixr2/nlmixr2#53]
In the category of "why would you do that, Bill":  When I tried to use `ini()` without loading `rxode2` first, it couldn't find `lotri()`.

``` r
model <- function () {
  description <- "One compartment PK model with linear clearance"
  ini({
    lka <- 0.45
    label("Absorption rate (Ka)")
    lcl <- 1
    label("Clearance (CL)")
    lvc <- 3.45
    label("Central volume of distribution (V)")
    prop.err <- c(0, 0.5)
    label("Proportional residual error (fraction)")
  })
  model({
    ka <- exp(lka + etalka)
    cl <- exp(lcl)
    vc <- exp(lvc)
    linCmt() ~ prop(prop.err)
  })
}

nlmixr2::ini(model, etalka~1)
#> Error in lotri({: could not find function "lotri"
library(rxode2)
#> rxode2 2.0.7 using 8 threads (see ?getRxThreads)
#>   no cache: create with `rxCreateCache()`
#> 
#> Attaching package: 'rxode2'
#> The following object is masked _by_ '.GlobalEnv':
#> 
#>     model
nlmixr2::ini(model, etalka~1)
#> ℹ promote `etalka` to between subject variability with initial estimate 1
#> ℹ change initial estimate of `etalka` to `1`
#>  ── rxode2-based solved PK 1-compartment model with first-order absorption ────── 
#>  ── Initalization: ──  
#> Fixed Effects ($theta): 
#>      lka      lcl      lvc prop.err 
#>     0.45     1.00     3.45     0.50 
#> 
#> Omega ($omega): 
#>        etalka
#> etalka      1
#>  ── Model (Normalized Syntax): ── 
#> function() {
#>     description <- "One compartment PK model with linear clearance"
#>     ini({
#>         lka <- 0.45
#>         label("Absorption rate (Ka)")
#>         lcl <- 1
#>         label("Clearance (CL)")
#>         lvc <- 3.45
#>         label("Central volume of distribution (V)")
#>         prop.err <- c(0, 0.5)
#>         label("Proportional residual error (fraction)")
#>         etalka ~ 1
#>     })
#>     model({
#>         ka <- exp(lka + etalka)
#>         cl <- exp(lcl)
#>         vc <- exp(lvc)
#>         linCmt() ~ prop(prop.err)
#>     })
#> }
```

<sup>Created on 2022-09-15 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

PR coming shortly.

**Comment by billdenney** [comment]
Shoot.  Sorry about that

**Comment by mattfidler** [comment]
Its OK, still an issue that needs to be fixed somehow.

**Comment by mattfidler** [comment]
Should be fixed by #54
